src/Aspire.Hosting.Azure.ServiceBus/AzureServiceBusExtensions.cs

```diff
@@ -233,20 +232,10 @@ public static IResourceBuilder<AzureServiceBusResource> RunAsEmulator(this IReso
             return builder;
         }
 
-        // Create a default file mount. This could be replaced by a user-provided file mount.
-        var configHostFile = Path.Combine(Directory.CreateTempSubdirectory("AspireServiceBusEmulator").FullName, "Config.json");
-
-        var defaultConfigFileMount = new ContainerMountAnnotation(
-                configHostFile,
-                AzureServiceBusEmulatorResource.EmulatorConfigJsonPath,
-                ContainerMountType.BindMount,
-                isReadOnly: true);
-
-        builder.WithAnnotation(defaultConfigFileMount);
-
         // Add emulator container
 
-        var password = PasswordGenerator.Generate(16, true, true, true, true, 0, 0, 0, 0);
+        // The password must be at least 8 characters long and contain characters from three of the following four sets: Uppercase letters, Lowercase letters, Base 10 digits, and Symbols
+        var passwordParameter = ParameterResourceBuilderExtensions.CreateDefaultPasswordParameter(builder.ApplicationBuilder, $"{builder.Resource.Name}-sqledge-pwd", minLower: 1, minUpper: 1, minNumeric: 1);
```

Comment:
```suggestion
        var passwordParameter = ParameterResourceBuilderExtensions.CreateDefaultPasswordParameter(builder.ApplicationBuilder, $"{builder.Resource.Name}-sql-pwd", minLower: 1, minUpper: 1, minNumeric: 1);
```

Do we need to explicitly call this "sqledge"? What if we changed it in the future?

Reply:
But the resource is named `$"{builder.Resource.Name}-sqledge"`, that's why it's named `$"{builder.Resource.Name}-sqledge-pwd"` with the extra `-pwd`. If we were to change the type of resource, and the name, we could also use a different one for the password.

Or we can name everything without `sqledge` at all.

Reply:
I think let's drop the `edge` in all for sure. I think `sql` is fine, but if you want to call it something more generic like `-data` I wouldn't object.